### ISSUE TITLE
Permission Nodes help

### ISSUE THREAD

**Issue by Derp1234567** [IntellectualSites/PlotSquared#1804]
I have a minecraft spigot server and it has plots and skyblock but if your not opped and you try and do /plot claim it says you dont have the permission node. I have looked up ways to fix it but none of them work. Please help

**Comment by RedstoneFuture** [comment]
Can you get more informations, please? E.g. the `/plot debugpaste` link and your permission.yml (!) ?

**Comment by braceyourself07** [comment]
https://github.com/IntellectualSites/PlotSquared/wiki/Commands-and-Permissions

**Comment by Derp1234567** [comment]
groups:
  default: true
    permissions:
    - modifyworld.*
    - plots.use
    - plots.auto
    - plots.claim
    - plots.buy
    - plots.visit
options:
      default: true
schema-version: 1

I have done /pex user <username> add plots.auto and it still doesnt work. I have permissionsex and i put that in there and i put it in the permissions.yml in my server that comes with it. I want everybody to have that so i put that as default. When i do the pex command nothing happens in the chat and i don't have the permissions. Only Ops have it and when i deop myself it says i need the permission node plots.auto

**Comment by SART-Jack** [comment]
Same issue, I assume it started when I upgraded to dev build 543 (file PlotSquared-Bukkit-17.12.09-59557cf-1945) (however I can't be sure, I only noticed the issue when two new players got frustrated that they couldn't claim new plots and left the server).

I downgraded to the previous version I was using, PlotSquared-Bukkit-17.09.13-ce90b36-1910 (no idea what build that is, sorry), however, it appears the issue is persistent.

We had a functional server and no changes were made to permissions, so it appears to be PlotSquared. Nonetheless, I'm taking a look at our permissions database right now.

**Comment by MCAlagaesia** [comment]
Same issue, I assume it started when I upgraded to dev build 543 (file PlotSquared-Bukkit-17.12.09-59557cf-1945) (however I can't be sure, I only noticed the issue when two new players got frustrated that they couldn't claim new plots and left the server).

I downgraded to the previous version I was using, PlotSquared-Bukkit-17.09.13-ce90b36-1910 (no idea what build that is, sorry), however, it appears the issue is persistent.

We had a functional server and no changes were made to permissions, so it appears to be PlotSquared. Nonetheless, I'm taking a look at our permissions database right now.

**Comment by MCAlagaesia** [comment]
Well, I'm stumped. In my build_permissions database, I found only one PlotSquared related permission: _plots.plot.10_ given to the _builder_ rank. Yet I'm 100% certain that we had a fully functional PlotSquared installation only days before.

I tried giving the _default_ user the following permissions. I tested whether a _default_ account could execute "/p auto" each time.

plots.auto
plots.plot.5
plots.use 
plots.permpack.basic

None worked. In desperation, I gave _default_ access to _plots.*_. That worked (not that it's a solution, I immediately removed the permission after testing it).


(I should note that upon rereading the original issue that we don't actually run Skyblock - I just inferred that Derp1234567 was also running build 543 based on the date the issue was opened. My apologies.)
